server/src/main/java/org/opensearch/identity/IdentityService.java

```diff
@@ -56,4 +63,41 @@ public Subject getSubject() {
     public TokenManager getTokenManager() {
         return identityPlugin.getTokenManager();
     }
+
+    public static IdentityService getInstance() {
```

Comment:
@dbwiddis @saratvemulapalli @reta  Do you have suggestions on how to do this better that is opted in the codebase?  static instance methods are functional but hard to use with dependency injection

Reply:
I would be curious to get thoughts here as well. I don't know how to handle this like you mentioned. It causes issues with testing as well but there is not an alternative way to fetch the service. The other services seem to largely live inside Node.java or similar.